### PR TITLE
Feature/update event types

### PR DESCRIPTION
**Pull request by jniravel**
#What Changed?
-  Added PayloadSuccessType and PayloadEventType for parsing widget event type
- Added WidgetEventType for the widget event callback type

#What to test?
- Perfom basic sanity testing

### CODE REVIEW

File: bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/StreamEvent.kt [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+        fun fromEvent(event: String): WidgetEventType? {
+            return values().find { it.event == event }
+        }
+    }
+}
+
+data class PayloadSuccessType(
+    val user_id: String,
+    val user_password: String,
+    val ClientName: String,
```
Comment (mark-aeropay) [inline bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/StreamEvent.kt:18]: mixing casings :/

**Review by mark-aeropay — APPROVED** [review]
looks good

File: bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/WebAppInterface.kt [hunk 2]
```diff
@@ -1,25 +1,57 @@
...
+        if(response.isEmpty) return
+        var widgetEventType: WidgetEventType? = null;
+        try{
+            widgetEventType = WidgetEventType.fromEvent(response.get("type").getAsString())!!
+        } catch (e: Exception) {
+            eventListener.onError("Invalid widget event type: $e", mContext)
+        }
+        if(widgetEventType == null) return
+        try {
+            when (widgetEventType) {
```
Comment (mark-aeropay) [inline bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/WebAppInterface.kt:29]: 👍

File: bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/StreamEvent.kt (hunk 1 above)
Comment (jniravel) [reply bank-link-sdk/src/main/java/com/aerosync/bank_link_sdk/StreamEvent.kt:18]: @mark-aeropay : We have fixed the casing for the 2.0 version-  AeroPass flow.

**Pull request merged**